packages/aws-cdk-lib/aws-ecs-patterns/lib/base/scheduled-task-base.ts

```diff
@@ -112,7 +112,7 @@ export interface ScheduledTaskImageProps {
   readonly logDriver?: LogDriver;
 
   /**
-   * The amount (in GiB) of ephemeral storage to be allocated to the task. The maximum supported value is 200 GiB.
+   * The amount (in GiB) of ephemeral storage to be allocated to the task. The maximum supported value is200 GiB.
```

Comment:
nit: typo

Reply:
Updated already. Which can find in the latest commit